### PR TITLE
Set and print bootnode ENR

### PR DESCRIPTION
**Pull request by cemozerr**
## PR Description
Set and print boot node ENR. Fixes [BC-431]

[BC-431]: https://pegasys1.atlassian.net/browse/BC-431

### CODE REVIEW

File: networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/DiscoveryNetwork.java [hunk 1]
```diff
@@ -61,6 +64,12 @@
     this.p2pNetwork = p2pNetwork;
     this.discoveryService = discoveryService;
     this.connectionManager = connectionManager;
+    initialize();
+  }
+
+  public void initialize() {
+    setBootNodeForkInfo();
```
Comment (mbaxter) [inline networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/DiscoveryNetwork.java]:
```suggestion
    setPreGenesisForkInfo();
```

File: networking/p2p/src/test/java/tech/pegasys/teku/networking/p2p/DiscoveryNetworkTest.java [hunk 2]
```diff
@@ -88,7 +91,7 @@ public void shouldStopConnectionManagerBeforeNetworkAndDiscovery() {
     discoveryNetwork.stop();
 
     verify(connectionManager).stop();
-    verifyNoInteractions(p2pNetwork, discoveryService);
```
Comment (mbaxter) [inline networking/p2p/src/test/java/tech/pegasys/teku/networking/p2p/DiscoveryNetworkTest.java:91]: May be worth verifying the expected `discoveryService` interaction, then asserting `verifyNoMoreInteractions(discoveryService)`.

**Review by mbaxter — APPROVED** [review]
LGTM

File: networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/DiscoveryNetwork.java (hunk 1 above)
Comment (cemozerr) [reply networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/DiscoveryNetwork.java]: done.

File: networking/p2p/src/test/java/tech/pegasys/teku/networking/p2p/DiscoveryNetworkTest.java (hunk 2 above)
Comment (cemozerr) [reply networking/p2p/src/test/java/tech/pegasys/teku/networking/p2p/DiscoveryNetworkTest.java:91]: good idea!

**Pull request merged**